UI/cypress/e2e/mobile.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Mobile view page tests
+  
+    Scenario: Subject list is bullet pointed on search reaults page in mobile view
+        Given a user has started the 'Find a tuition partner' journey
```

Comment:
These can be simplified to:

```
Given a user has arrived on the 'Search results' page
And a user is using a 'phone' 
Then the subject list is bullet pointed
```

This removes the need for the mobile specific `the search result page is displayed` step.

Changing the TP details pages to something similar to:

```
Given a user has arrived on the 'Tuition Partner' page for 'connex-education-partnership'
And a user is using a 'phone'
Then the subject list is bullet pointed
```

Would also somewhat simplify them

Reply:
Changed as suggested